bin/windows-env.ps1

```diff
@@ -27,13 +27,19 @@ $mingwThreads = "win32"
 if ($arch -eq 64) {
   $mingwExceptions = "seh"
   $mingwArch = "x86_64"
+  $opensslArch = "x64"
+  $rubyArch = "x64"
 } else {
   $mingwExceptions = "sjlj"
   $mingwArch = "i686"
+  $opensslArch = "x86"
+  $rubyArch = "i386"
 }
 $mingwVer = "${mingwArch}_mingw-w64_${mingwVerNum}_${mingwThreads}_${mingwExceptions}"
 
-$opensslPkg = "openssl-1.0.0s-x64-windows"
+$rubyPkg = "ruby-2.1.7-${rubyArch}-mingw32"
+
+$opensslPkg = "openssl-1.0.2e-${opensslArch}-windows"
```

Comment:
@Iristyle the issue was with OpenSSL 1.0.0s (see discussion in https://tickets.puppetlabs.com/browse/PUP-5215)